server/create_tiff.py

```diff
@@ -30,6 +30,8 @@
 
 convert_command = (
     'vips',
+    # '--vips-concurrency', '1',
+    # '--vips-progress',
```

Comment:
I did, as optionally enabling them has helped with tracking down vips issues on certain installations.  I should add a comment to that effect.